index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+// Create PDO connection (inc set fetch mode)
+$db = new PDO ('mysql:host=db; dbname=milo-collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+// Write SQL query to retrieve all collection items
+$query = $db->prepare("SELECT `name`, `age`, `instrument`, `band`, `technical-prowess`, `image` FROM `idols`;");
+$query->execute();
+$idols = $query->fetchAll();
+
+// Loop over results to echo out stats for each item
+function displayItems($idols) {
```

Comment:
Also, if you are going to use a function, it's worth making sure you include type hints.